Anytype/Sources/PresentationLayer/Common/UIKit/Pickers/FilePicker/CommonViews+Pickers+File+UIKit.swift

```diff
@@ -40,8 +40,8 @@ extension UIKitPickerViewController {
     }
 }
 
-// MARK: - Picker
-class Picker: UIViewController {
+// MARK: - AnytypePicker
+class AnytypePicker: UIViewController {
```

Comment:
Rename our custom `Picker` to `AnytypePicker` because of naming conflict with SwiftUI component